src/__tests__/outer-form-submit.test.tsx

```diff
@@ -6,7 +6,7 @@ import { render } from '@testing-library/react';
 import { getRequiredPropsForComponent } from './required-props-for-components';
 import { getAllComponents, requireComponent } from './utils';
 
-const skippedComponents = ['button'];
+const skippedComponents = ['button', 'toggle-button'];
```

Comment:
Why is this change necessary?

This test was introduced to make sure that there was no element with `type="submit"` which would accidentally cause form submission. The button component can render that attribute in the native button if using `formAction="submit"`, but the new ToggleButton component doesn't use `formAction="submit"` in the InternalButton props so it should not have that issue. So is there any reason to not cover this new component here along with the rest of components?

Reply:
Fair enough, removed it